user/customizing-the-build.md

```diff
@@ -523,12 +524,27 @@ jobs:
 ```
 {: data-file=".travis.yml"}
 
+#### Conditionally Allowing Jobs to Fail
+
+Allowed failures can include a [condition](https://docs.travis-ci.com/user/conditional-builds-stages-jobs#conditionally-allowing-jobs-to-fail) using the key `if`.
+
+For example, the following would allow the job using `rvm: 1.9.3` to fail
```

Comment:
Nothing technical, but mentioning 1.9.3 in 2020 seems really outdated.

Reply:
true, but i don't think i want to change all the examples everywhere with this change :)